Tasks/Xcode/task.json

```diff
@@ -301,6 +301,10 @@
         "FailedToGenerateExportOptionsPlist": "Failed to automatically generate an `Export options` plist file to export an IPA. Please specify an `Export options` plist file in `Package options`.",
         "LocateXcodeBasedOnVersion": "Locating the developer path for Xcode `%s`",
         "FailedToLocateSpecifiedXcode": "Unable to resolve the developer path for Xcode `%s`. Set the `%s` environment variable on the agent machine, or set `Xcode version` to `Default` or `Specify path`.",
-        "CantDetermineProvisioningStyle": "The provisioning style was not found in a pbxProject file. Automatic signing will be used. If the project uses manual signing, auto-export may fail. As a workaround, `Export options` can be set to `Plist` or `Specify`."
+        "CantDetermineProvisioningStyle": "The provisioning style was not found in a pbxProject file. Automatic signing will be used. If the project uses manual signing, auto-export may fail. As a workaround, `Export options` can be set to `Plist` or `Specify`.",
+        "MultipleSchemesFound": "The workspace contains multiple schemes. The scheme must be specified.",
```

Comment:
When there are multiple workspaces, we choose the first one and warn. We could sort and choose the first scheme, but I worry a bit about the build building something else if they later add another scheme. Given multiple schemes, I think it would be better to lock to one of them in the build definition.